config/config.ini.example

```diff
@@ -85,42 +84,6 @@
 #geofence-excluded-file:        # File to define forbidden areas inside scan area. Regarded this as inverted geofence. Can be combined with geofence-file.
 
 
-# Search settings
-#################
-
-#china                          # Coordinates transformer for China.
-#no-jitter                      # Don't apply random -9m to +9m jitter to location.
-#altitude:                      # Default altitude in meters. Retrieve this value regarding your scan location from the web. (default=507)
-#altitude-variance:             # Variance for --altitude in meters (default=1)
-#use-altitude-cache             # Query the Elevation API for each step, rather than only once, and store results in the database.
-                                # Make sure your Google Elevation API is enabled
-#workers-per-hive:              # Only referenced when using --beehive. Sets number of workers per hive. (default=1)
-#workers:                       # Number of search worker threads to start. (default=#accounts)
-#spawn-delay:                   # Number of seconds after spawn time to wait before scanning to be sure the Pokemon is there. (default=10)
-#kph:                           # Set a maximum speed in km/hour for scanner movement. 0 to disable.  (default=35)
-#bad-scan-retry:                # Number of bad scans before giving up on a step. (default=2, 0 to disable)
-#skip-empty                     # Enables skipping of empty cells in normal scans - requires previously populated database. (not to be used with -ss)
-#min-seconds-left:              # Time that must be left on a spawn before considering it too late and skipping it. (default=0)
-#gym-info                       # Enables detailed gym info collection. (default=False)
```

Comment:
This argument is used by the front end, so it should not be removed. It enables the gym sidebar.

Reply:
`gym_info` should not be used for this I think. I'd rather introduce a new parameter to  enable the sidebar, or just show the settings permanently. This paramter has been used in the search worker only. There's no other place wher it is being used.